newIDE/app/src/AssetStore/AssetsHome.js

```diff
@@ -107,7 +112,7 @@ export const AssetsHome = ({
                         color="primary"
                         size="body2"
                       >
-                        {pack.assetsCount} Assets
+                        {pack.assetsCount} <Trans>Assets</Trans>
```

Comment:
You can write `<Trans>{pack.assetsCount} Assets</Trans>`. It can give more context to translators